### PR TITLE
ci: Fix CODEOWNERS due to team split

### PR DESCRIPTION
**Pull request by ddyurchenko**
### What does this PR do?

The team #apm-reliability-and-performance was split. This PR updates handle and assigns correct codeowners for benchmarking Gitlab CI pipelines, owned by APM Ecosystems Performance.

### Motivation

Fix handle after team split.

### CODE REVIEW

**Review by FlorentClarret — APPROVED** [review]

File: tasks/libs/pipeline/github_slack_map.yaml
Comment (CelianR) [inline tasks/libs/pipeline/github_slack_map.yaml:47]: If there is a split, shouldn't this be two lines containing two teams?

**Review by CelianR — APPROVED** [review]

File: tasks/libs/pipeline/github_slack_map.yaml
Comment (ddyurchenko) [reply tasks/libs/pipeline/github_slack_map.yaml:47]: Thanks for question Celian!
No, the change is correct. After split, only my team owns CI code of benchmarking CI job for trace-agent.
#apm-ecosystems-reliability (the other team) doesn't own anything in datadog-agent repo as of now.

Comment (ddyurchenko) [reply tasks/libs/pipeline/github_slack_map.yaml:47]: I can't merge the changes though (see "Merging is blocked" in Github). @CelianR could you please click merge button? :pray: :bow:

**Pull request merged**
